libs/wire-api/src/Wire/API/Team/Feature.hs

```diff
@@ -212,60 +222,48 @@ newtype TeamFeatureStatusNoConfig = TeamFeatureStatusNoConfig
   { tfwoStatus :: TeamFeatureStatusValue
   }
   deriving newtype (Eq, Show, Generic, Typeable, Arbitrary)
+  deriving (ToJSON, FromJSON, S.ToSchema) via (Schema TeamFeatureStatusNoConfig)
 
+-- TODO: Delete
 modelTeamFeatureStatusNoConfig :: Doc.Model
 modelTeamFeatureStatusNoConfig = Doc.defineModel "TeamFeatureStatusNoConfig" $ do
-  Doc.description $ "Configuration for a team feature that has no configuration"
+  Doc.description "Configuration for a team feature that has no configuration"
```

Comment:
```suggestion
  Doc.description "Team feature that has no configuration beyond the boolean on/off switch."
```

(I know it has been like this, and I may have written this, but I think this is a good time to fix that, too.)

(Come to think of it should this be added to the schema?)

Reply:
I removed these Swagger 1.2 entries in the module.